src/execCmd.ts

```diff
@@ -22,6 +24,10 @@ export interface ExecCmdOptions extends ExecOptions {
 }
 
 export interface ExecCmdResult {
+  /**
+   * The command
+   */
+  command: string;
```

Comment:
What's the usecase for this?

Reply:
I would like to be able to see what the command was, specifically when looking at setup results. See my example in [comment](https://github.com/salesforcecli/cli-plugins-testkit/pull/30#discussion_r580491391).